pomdp.py

```diff
@@ -163,19 +164,36 @@ def __get_value(self, i):
         return i + 1
 
     def __get_states(self, i):
-        # TODO support states as number
         line = self.contents[i]
         self.states = line.split()[1:]
+        if len(self.states) == 1 and unicode(self.states[0], 'utf-8').isnumeric():
```

Comment:
Two thoughts:

1. Can we pull this check into a function like `is_numeric` so that we can reuse it for the different checks (and fix edge cases as they pop up)?

2. Instead of requiring the document to be `utf-8`, could we instead just call `int()` on the string, wrapped in a `try` block? I know that works on py3, but can't remember for py27.